pkg/agent/flags/keys/keys.go

```diff
@@ -47,6 +47,7 @@ const (
 	LogIngestServerURL         FlagKey = "log_ingest_url"
 	TraceIngestServerURL       FlagKey = "trace_ingest_url"
 	DisableTraceIngestTLS      FlagKey = "disable_trace_ingest_tls"
+	DisableOsqueryHealthchecks FlagKey = "disable_osquery_healthchecks"
```

Comment:
Thoughts about whether should do this as `DisableOsqueryHealthchecks` vs `PowersaveMode`?

Reply:
I had gone back and forth on similar too, thinking about this versus something like `InModernStandby`. I ended up on this one because the osquery extension doesn't need to know why it should skip healthchecks, just what to do -- but I do not feel that strongly about it.